lua/minigames/minigames/minigame_base/shared.lua

```diff
@@ -58,4 +65,20 @@ if SERVER then
 	function MINIGAME:IsSelectable()
 		return true
 	end
+else
+	---
+	-- This function is called as soon as the @{MINIGAME} is activated and should be used to display a EPOP message informing the player about the activation
+	-- @realm client
+	function MINIGAME:ShowActivationEPOP()
+		if not istable(self.lang) or not EPOP then return end
+
+		EPOP:AddMessage({
+				text = LANG.TryTranslation("ttt2_minigames_" .. self.name .. "_name"),
+				color = COLOR_ORANGE
+			},
+			self.lang.desc and LANG.TryTranslation("ttt2_minigames_" .. self.name .. "_desc") or nil,
+			12,
+			true
+		)
```

Comment:
```suggestion
		EPOP:AddMessage({
				text = LANG.TryTranslation("ttt2_minigames_" .. self.name .. "_name"),
				color = COLOR_ORANGE
			},
			self.lang.desc and LANG.TryTranslation("ttt2_minigames_" .. self.name .. "_desc") or nil,
			12,
			nil,
			true
		)
```
this should work (untested)
also i don't know why i didn't stumble upon this error when i did #5 ...